server/src/main/java/org/elasticsearch/action/main/MainResponse.java

```diff
@@ -69,6 +73,8 @@ public String getClusterUuid() {
         return clusterUuid;
     }
 
+    public String getClusterDynamicName() { return clusterDynamicName; }
```

Comment:
I think Javadoc would be nice for these, especially now that we are using them in the high level rest client.